docs/overview.md

```diff
@@ -0,0 +1,844 @@
+argument.
+
+```sql
+SELECT SUBSTRING(ProductSold, CHAR_LENGTH(ProductSold) - 2, 2)
+FROM Transactions;
+```
+
+Combines SUBSTRING() with CHAR_LENGTH() to get characters from the end of the string.
+
+#### Date and Time Functions
+
+##### DATETRUNC()
+
+Truncate a timestamp to a specified unit:
+
+```sql
+SELECT DATETRUNC(DAY, saleDate)
+FROM Sales;
+```
+
+Supported Units: YEAR, MONTH, DAY, HOUR, MINUTE, SECOND, WEEK, DAY_OF_YEAR, ISO_WEEK, ISO_WEEKDAY.
+
+##### DATEADD()
+
+Add an interval to a timestamp:
+
+```sql
+SELECT DATEADD(YEAR, 1, saleDate), saleDate
+FROM Sales;
+```
+
+##### DATEDIFF()
+
+Calculate the difference between two timestamps:
+
+```sql
+SELECT DATEDIFF(YEAR, CURRENT_TIMESTAMP, saleDate), saleDate
+FROM Sales;
+```
+
+##### EXTRACT()
+
+Extract a part of a timestamp:
+
+```sql
+SELECT EXTRACT(YEAR FROM saleDate), saleDate
+FROM Sales;
+```
+
+##### CASTING TO/FROM DATE, TIMESTAMP
+
+```sql
+SELECT CAST(EXTRACT(YEAR FROM saleDate) AS integer), saleDate
+FROM Sales;
+
+SELECT CAST('1975-01-23' AS TIMESTAMP) AS Birthdate, saleDate
+FROM Sales;
+```
+
+Note: MongoDB supports only the TIMESTAMP type.
+
+##### CURRENT_TIMESTAMP
+
+Get the current timestamp:
+
+```sql
+SELECT CURRENT_TIMESTAMP
+FROM Sales;
+```
+
+##### ISO_WEEKDAY
+
+Get the ISO day of the week:
+
+```sql
+SELECT EXTRACT(ISO_WEEKDAY FROM saleDate), saleDate
+FROM Sales;
+```
+
+#### Numeric Functions
+
+##### TO/FROM EPOCH
+
+To Epoch:
+
+```sql
+SELECT CAST(saleDate AS LONG)
+FROM Sales;
+```
+
+From Epoch:
+
+```sql
+SELECT CAST(epochValue AS TIMESTAMP)
+FROM SomeTable;
+```
+
+#### Unsupported Functions
+
+- SIMILAR TO
+- RANDOM
+- Timezone Conversion
+Not Supported: MongoDB stores dates in UTC.
+- GROUP_CONCAT
+
+### Additional Notes
+
+- Polymorphic Schemas:
+Be cautious when using aggregate functions on fields with polymorphic schemas or non-comparable types like documents and arrays.
+The term "polymorphic" schema is used to refer to a field/column that can have multiple types, e.g. int and string.
+
+- Escape Characters:
+Use the ESCAPE clause to specify custom escape characters in LIKE patterns.
+
+- Alias Assignments:
+Required when using nested selects or derived tables.
+
+- Date Functions:
+MongoSQL supports various date functions, but only the TIMESTAMP data type is available.
+
+## Additional Features
+
+### Security Features
+
+The MongoDB Atlas SQL ODBC driver supports all authentication mechanisms
+supported by MongoDB (x509, OAuth, LDAP, etc...). See [authentication mechanisms](https://www.mongodb.com/docs/manual/core/authentication/#authentication-mechanisms)
+for a full list of supported authentication mechanisms.
+
+If [configured](https://www.mongodb.com/docs/manual/tutorial/configure-ssl/), the MongoDB Atlas SQL ODBC driver supports TLS/SSL connections.
+
+### Logging and Diagnostics
+
+By default, the MongoDB Atlas SQL ODBC driver produces logs in `/%HOME%/MongoDB/Atlas SQL/logs`.
```

Comment:
This is for linux.  On windows it looks like we try to set it to:
```
C:\Users\{user}\Documents\MongoDB\Atlas SQL ODBC\{driver_log_version}\logs
```
Would be useful to add.